src/main/java/tech/fastj/systems/behaviors/package-info.java

```diff
@@ -0,0 +1,5 @@
+package tech.fastj.systems.behaviors;
+
+/**
+ * Code for the FastJ behaviour system.
```

Comment:
Consider: "API for FastJ's behavior system."

> behaviour

Furthermore, please use American-style wording for any documentation/code in FastJ.